benchmarks/src/test/java/org/apache/druid/benchmark/indexing/IndexPersistBenchmark.java

```diff
@@ -87,101 +90,95 @@
   @Param({"true", "false"})
   private boolean rollup;
 
-  @Param({"none", "moderate", "high"})
-  private String rollupOpportunity;
+  @Param({"0", "1000", "10000"})
+  private int rollupOpportunity;
```

Comment:
Sorry I missed this earlier, but I feel that we should retain the textual values for rollupOpportunity as that is more user-friendly when reading the benchmark results.

Reply:
OK. I rolled back this change.